Documentation/install/openstack/openstack-terraform.md

```diff
@@ -66,7 +66,23 @@ Download the Tectonic Terraform modules.
 
 ```bash
 $ terraform init platforms/openstack/<flavor>
-```
+Downloading modules...
+Get: modules/openstack/<flavor>/vpc
```

Comment:
actual output or presumed? Curious if every platform would actually have a `vpc` module, for instance

Reply:
wasn't tested on all platforms - only linux (Kyle) and macOS (Rob). we should run the others and see.

Reply:
to produce output for a provider:
```
$ curl -O .../tectonic_1.7.5-tectonic.1-rc.6.tar.gz
$ tar xzf tectonic_1.7.5-tectonic.1-rc.6.tar.gz
$ cd tectonic
$ export PATH={$pwd}/tectonic-installer/linux:$PATH
$ terraform init platforms/azure
```

Reply:
actually, what worked for me was:
```
curl -O https://releases.tectonic.com/candidates/tectonic_1.7.5-tectonic.1-rc.6.tar.gz
tar xzvf tectonic_1.7.5-tectonic.1-rc.6.tar.gz
cd tectonic
export PATH=$PATH:$(pwd)/tectonic-installer/linux
terraform init platforms/<platform>
```

Reply:
https://gist.github.com/joshix/704b612876a225360db8fa3372612a11

Actual output for each platform.

Reply:
You want to prepend rather than append to the PATH because if you have a stock terraform on your regular PATH, it will resolve before the terraform you're trying to add to the path.

so something like:
`export PATH=$(pwd)/tectonic-installer/linux:$PATH`

Reply:
thank you - i can check your returns against mine.

Reply:
azure does not match. all the others do. I'll run it again, and see what happens.

Reply:
that one matched. updated.